rln/src/public.rs

```diff
@@ -262,26 +122,3 @@ impl Default for RLN {
         Self::new()
     }
 }
-
-// NOTE: Expensive test, ignoring by default
-#[ignore]
-#[test]
-fn rln_proof() {
```

Comment:
This test was removed, I can't find the equivalent test that replicates this

Reply:
You're right, is in the main now, will move.